packages/third-parties/src/google/index.tsx

```diff
@@ -1,2 +1,3 @@
 export { default as GoogleMapsEmbed } from './GoogleMapsEmbed'
```

Comment:
Like we discussed, can you verify that `'use client'` isn't needed for these exports to work in server components per the [docs](https://nextjs.org/docs/app/building-your-application/rendering/composition-patterns#using-third-party-packages-and-providers:~:text=To%20fix%20this%2C%20you%20can%20wrap%20third%2Dparty%20components%20that%20rely%20on%20client%2Donly%20features%20in%20your%20own%20Client%20Components%3A)? Ideally, I wouldn't want developers to deal with adding `'use client'` themselves if we can/should do it here directly.

(We can deal with this in a future PR)

Reply:
Also we can deal with this on a case-by-case basis. For example, if certain utilities (like `useGoogleTagManager`) must be used in Client components, than that's fine. But we probably want to make it clear which ones have to be used in client components versus others that can be used in both server + client (with `use client` included within them)

Reply:
Thats a good point. I'm not very sure. @huozhi @gnoff Do you have any thoughts about adding `use client` to the third party components

Reply:
If a file exports a React client component, it should add the `use client` directive.

If a file exports a React Hooks, it should add the `import 'client-only'`.

Reply:
Yeah I think we need to mark this file with `'use client'`.

Reply:
Thanks for that info!